package.json

```diff
@@ -65,7 +58,13 @@
     "react": "^15.1.0",
     "react-bootstrap": "^0.30.0",
     "react-dom": "^15.1.0",
-    "select2": "~3.5.2-browserify"
+    "select2": "~3.5.2-browserify",
+    "react-redux": "^5.0.2",
+    "redux": "^3.6.0",
+    "redux-logger": "^2.8.1",
+    "redux-mock-store": "^1.2.2",
+    "redux-thunk": "^2.2.0",
+    "seamless-immutable": "^7.0.1"
```

Comment:
Could you please order these alphabetically? Some npm versions automatically do that, just to prevent the next one touching this file from changing it in an unrelated commit.